libs/utils/trace.py

```diff
@@ -292,20 +293,22 @@ def __loadTasksNames(self, tasks):
         """
         Try to load tasks names using one of the supported events.
 
-        :param tasks: list of task names
-        :type tasks: list(str)
+        :param tasks: list of task names. If None, load all tasks found.
+        :type tasks: list(str) or NoneType
         """
+        def load(tasks, event, name_key, pid_key):
+            df = self._dfg_trace_event(event)
+            if tasks is None:
```

Comment:
Can we split this patch into a refactoing one which adds the load method and a "Populate all task names when no tasks passed" which adds this "if task is None:" bits?